modules/seqexec/server/src/main/java/edu/gemini/seqexec/server/altair/LgsSfoControl.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package edu.gemini.seqexec.server.altair;
```

Comment:
It has to be Java because I am using Java code to connect to EPICS, which requires a Java `Enum` to conect to a enum EPICS channel. That is in the **acm** module.